### ISSUE TITLE
`swupd repair` unable to repair broken symlink?

### ISSUE THREAD

**Issue by ahkok** [clearlinux/swupd-client#1062]
```
$ sudo swupd repair
...
Repairing corrupt files
 -> Hash mismatch for file: /usr/lib64/libjli.so -> fixed
	...100%
...
```

Repeating the command shows the same error occurs every time.

Possible root cause: 
```
$ readlink /usr/lib64/libjli.so
libjli11.so
$ stat /usr/lib64/libjli1.so
stat: cannot stat '/usr/lib64/libjli1.so': No such file or directory
```

The item is a broken symlink. Tracking back to the actual package, it clearly shows the link is properly made in the spec file:
```
openjdk/openjdk.spec:ln -s  /usr/lib/jvm/java-1.8.0-openjdk/jre/lib/amd64/jli/libjli.so %{buildroot}/usr/lib64/libjli.so
```

Fetching the RPM from download.clearlinux.org shows the contents are OK as well:
(From rpm2cpio -> `ls`)
```
lrwxrwxrwx   1 root     root           59 Aug  2 11:12 ./usr/lib64/libjli.so -> /usr/lib/jvm/java-1.8.0-openjdk/jre/lib/amd64/jli/libjli.so
```

**Comment by ahkok** [comment]
but it fixes `xemacs` just fine? weird.

**Comment by ahkok** [comment]
It seems to be able to fix other broken symlinks just fine, not this one.

**Comment by phmccarty** [comment]
@ahkok I opened an issue at `distribution` about this last week: https://github.com/clearlinux/distribution/issues/1093

**Comment by AthenasJimenez** [comment]
This is a ldconfig thing, it overwrites the symbolic link. So no, this is not an issue with swupd.

**Comment by phmccarty** [comment]
The issue with the openjdk packages has been resolved (see clearlinux/distribution#1093), so we can close this. It was not a swupd bug.
